index.d.ts

```diff
@@ -0,0 +1,22 @@
+declare module '@discoveryjs/json-ext' {
+    import { Readable } from 'stream';
+
+    export function parseChunked<T>(stream: Readable | Generator<any, void, unknown>): Promise<T>;
+
+    export function stringifyStream(value: any, replacer?: ((this: any, key: string, value: any) => any) | undefined, space?: string | number | undefined): Readable;
```

Comment:
lets move replacer function into separate type to make code more clearer )

Reply:
Replacer accept array of string and numbers either (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify)

Reply:
I've used https://github.com/microsoft/TypeScript/blob/main/lib/lib.es5.d.ts#L1059 and Fixed